### ISSUE TITLE
Unable to use command blocks 

### ISSUE THREAD

**Issue by Gunner76th** [TheCricket/FTB-Islands#38]
I am running a SkyFactory 2.5 server, and I was attempting to setup an auto island system similar to Infinity Evolved Skyblock. When I entered the command /island_create @p and then attempted to test it, the console in the command block returned that "You do not have permission to use this command" in the previous output on the command block. I was both op when I setup the command block, as well as when I pushed the button to trigger the command block, and even in creative mode. 

The version of FTP Islands that came with SkyFactory is 1.3.1, which according to Curse is the latest release. I have also seen that a request was made on the Curse page for command block functionality, and I was curious if this is being looked into, and if not, if this could be implemented. I am not sure why the command block is being denied the ability to execute the command. 

When I run the command /island_create Test and /island_delete Test, both of them execute perfectly fine with no errors, so I am seriously confused as to why the command block is returning a permissions error.

**Comment by Gunner76th** [comment]
![2016-12-03_00 12 12](https://cloud.githubusercontent.com/assets/5566872/20857001/37c25a2c-b8ed-11e6-81eb-4f0788102803.png)
Here is a screenshot of the Command Block output when I was testing the above. Also this appears to be a dupeplicate of issue https://github.com/Kolatra/FTB-Islands/issues/34 As I posted on that issue that is long since ben closed, were the changes posted on Oct 3 not compiled and pushed to Curse for some specific reason? I see where you stated that command blocks are now usable, however there has been no updated release on Curse since May.

**Comment by Gunner76th** [comment]
After digging into the repo, I also found an unresolved request on the commit regarding this specific code update and release. https://github.com/Kolatra/FTB-Islands/commit/abb8341954f51c02f61a1837de439d5bdf1a24f6#diff-a825e1bb4f41c73d44debdfcfb50e29e

And then again I see the pull request, and commit closure, https://github.com/Kolatra/FTB-Islands/pull/29 and yet no release with the updated code. As we are now 60 days past the code update, is this expected to be compiled and pushed to Curse soon, or is it possible to have this compiled and released in some other fashion so that as server admins we can begin using this mod and its functionality?

**Comment by kolatra** [comment]
New version of the mod is going on Curse with this fix implemented. Sorry about taking so long.

**Comment by Gunner76th** [comment]
Thank you so much. I was actually in the process of learning how to do  a gradle build. When I ran gradlew jar it kicked an error about diamond operators not being supported in 1.6 and to use -source 7 instead. I am new to java coding and compiling so learning by fire so to speak. I am sure I did something wrong, just not sure what it was I did. 

Again thanks for the quick response, it is greatly appreciated.
